src/data/info.json

```diff
@@ -15,4 +15,161 @@
     "title": "Reflection Pool",
     "text":
         "This pool goes by many names, ‘the reflection pond’, ‘the moat’, ‘the reflection pool’ or just the University ‘Pond’. It was designed by Rodney Alsop to enhance the beauty of Winthrop Hall by giving the impression of greater height. It was completed only just in time for the official opening of Winthrop Hall on April 13, 1932. This was largely owing to the efforts of the student body (men) who volunteered to provide the labour force if the University provided the material. The pond was completed just hours before the ceremony took place and filled with water even though the cement was still wet (it was later drained, and the concrete had to be redone)."
-}]
\ No newline at end of file
+},
+{
+    "id": 4,
+    "title": "Balconette on Administration Building",
+    "text":
+        ""
+},
+{
+    "id": 5,
+    "title": "Whitfeld Court",
+    "text":
+        ""
+},
+{
+    "id": 6,
+    "title": "Sunken Garden",
+    "text":
+        ""
+},
+{
+    "id": 7,
+    "title": "Rondels over the windows of the Administration Building",
+    "text":
+        ""
+},
+{
+    "id": 8,
+    "title": "Shann Memorial",
+    "text":
+        ""
+},
+{
+    "id": 9,
+    "title": "Sunken Garden",
```

Comment:
This is a duplicate marker. If you use the same marker twice, just use the same id and don't duplicate it in this file.